lib/handlers.js

```diff
@@ -76,10 +76,9 @@ module.exports = function (dataset, opts) {
 
     http({
       timeout: opts.timeout || DEFAULT_TIMEOUT,
-      guid: opts.guid,
+      url: opts.url + '/' + buildRoute(opts, dataset),
```

Comment:
If someone passes an `opts.url` with a trailing slash this might cause errors, e.g if `opts.url` is `acme.example.com/` you'd get `acme.example.com//dataset`. I'm not 100% sure, but best be safe.

Modifying `buildRoute` to use the `url` in `opts` should do the trick.